packages/datetime/src/dateRangeInput.tsx

```diff
@@ -463,6 +469,16 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
 
     // Key down
 
+    private handleStartInputKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+        this.handleInputKeyDown(e);
+        Utils.safeInvoke(this.props.startInputProps.onKeyDown, e);
+    }
+
+    private handleEndInputKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+        this.handleInputKeyDown(e);
+        Utils.safeInvoke(this.props.endInputProps.onKeyDown, e);
+    }
```

Comment:
i mean i can think of _ways_ of refactoring this, but i'm not sure they'd be strictly better. we can investigate later, maybe.